### PR TITLE
LWDEV-8718 Edge case with retry of StartCashoutCommand of the last ca…

### PR DESCRIPTION
**Pull request by KonstantinRyazantsev**
…shout in the batch has been fixed + tests have been added.

### CODE REVIEW

File: src/Lykke.Job.BlockchainCashoutProcessor.Core/Domain/Batching/CashoutsBatchAggregate.cs [hunk 1]
```diff
@@ -130,11 +130,14 @@ public static Guid GetNextId()
...
 
-        public TransitionResult AddCashout(Guid cashoutId, Guid clientId, string toAddress, decimal amount)
+        public TransitionResult AddCashout(BatchedCashoutValueType cashout)
         {
             var oldCashoutsCount = Cashouts.Count;
 
-            Cashouts.Add(new BatchedCashoutValueType(cashoutId, clientId, toAddress, amount));
+            if (oldCashoutsCount < CountThreshold)
+            {
+                Cashouts.Add(cashout);
```
Comment (some-dev) [inline src/Lykke.Job.BlockchainCashoutProcessor.Core/Domain/Batching/CashoutsBatchAggregate.cs:139]: Suggest to move:
`LastCashoutAdditionMoment = DateTime.UtcNow;` 
just after the 
`Cashouts.Add(cashout);` line 
and remove condition at 170

**Review by some-dev — APPROVED** [review]
